app/src/App.tsx

```diff
@@ -0,0 +1,73 @@
+import axios from "axios";
+import { useState } from "react";
+import "./App.css";
+
+import useRazorpay, { RazorpayOptions } from "react-razorpay";
+
+interface PayData {
+  amount: string;
+  currency: string;
+  id: string;
+}
+
+const App = () => {
+  const Razorpay = useRazorpay();
+  const [shoe, setShoe] = useState({
+    name: "Training Shoes",
+    creator: "Nike",
+    img: "https://images.pexels.com/photos/3490360/pexels-photo-3490360.jpeg?auto=compress&cs=tinysrgb&dpr=1&w=500",
+    price: 60000,
+  });
+
+  const initPay = (data: PayData) => {
+    const options: RazorpayOptions = {
+      key: "rzp_test_DPn50kPkUTWgOa",
+      amount: data.amount,
+      currency: data.currency,
+      name: shoe.name,
+      description: "Test",
+      image: shoe.img,
+      order_id: data.id,
+      handler: async (response: any) => {
```

Comment:
should this variable be name something else ? we don't post `response` to an API, if there's a specific term you can use -use that else generic variable can be called `requestBody`

Reply:
Pls see if `any` can be removed